simulation/sub8_gazebo/worlds/a_whole_new.world

```diff
@@ -130,6 +130,12 @@
       </link>
     </model>
 
+    <include>
+      <name>transdec_pinger</name>
+      <uri>model://pinger</uri>
+      <pose>15 0 -2 0 0 0 </pose>
```

Comment:
Totally random starting position here, please change to something more appropriate